sonaranalyzer-dotnet/build/rspec-templates/CSharpTestTemplate.cs

```diff
@@ -30,7 +30,8 @@ public class $DiagnosticClassName$Test
         [TestCategory("Rule")]
         public void $DiagnosticClassName$()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs", new $DiagnosticClassName$());
+            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs",
```

Comment:
Why is this required? Should we create a policy of adding all arguments with index>=1 on the next line?